pkg/kubelet/server_test.go

```diff
@@ -987,7 +987,7 @@ func TestServePortForward(t *testing.T) {
 
 			p, err := strconv.ParseUint(test.port, 10, 16)
 			if err != nil {
-				t.Fatalf("%d: error parsing port string '%s': %v", i, port, err)
+				t.Fatalf("%d: error parsing port string '%s': %v", i, test.port, err)
```

Comment:
Agreed, but there are least 70 more occurrences of this pattern in the code base. I don't mind fixing it, but I'd rather dedicate an entire PR to that one thing.